eks-distro-base/tests/run_tests.sh

```diff
@@ -110,14 +114,14 @@ function check_base-iptables() {
 }
 
 function check_base-csi() {
-    if docker run $IMAGE_REPO/eks-distro-minimal-base-csi:$IMAGE_TAG xfs_info -V | grep -v 'xfs_info version'; then
+    if docker run --pull=always $IMAGE_REPO/eks-distro-minimal-base-csi:$IMAGE_TAG xfs_info -V | grep -v 'xfs_info version'; then
```

Comment:
these are to eliminate the need to manually pull the latest pushed image from ECR before running the tests?

Reply:
Yeah without it if you built the image more than once the tests would still use the previously built image since it would be in the docker cache already.